doc/user/content/guides/cdc-mysql.md

```diff
@@ -0,0 +1,98 @@
+
+### Database Setup
+
+Before deploying a Debezium connector, you need to ensure that the upstream database is configured to support [row-based replication](https://dev.mysql.com/doc/refman/8.0/en/replication-rbr-usage.html). As _root_:
+
+1. Check the `log_bin` and `binlog_format` settings:
+
+    ```sql
+    SHOW VARIABLES
+    WHERE variable_name IN ('log_bin', 'binlog_format');
+    ```
+
+    For CDC, binary logging must be enabled and use the `row` format. If your settings differ, you can adjust the database configuration file (`/etc/mysql/my.cnf`) to use `log_bin=mysql-bin` and `binlog_format=row`. Keep in mind that changing these settings requires a restart of the MySQL instance and can affect database performance.
+
+    **Note:** Additional steps may be required if you're using MySQL on [Amazon RDS](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_LogAccess.MySQL.BinaryFormat.html).
+
+1. Grant enough privileges to the replication user to ensure Debezium can operate in the database:
+
+    ```sql
+    GRANT SELECT, RELOAD, SHOW DATABASES, REPLICATION SLAVE, REPLICATION CLIENT ON *.* TO "user";
+
+    FLUSH PRIVILEGES;
+    ```
+
+### Deploy Debezium
+
+Debezium is deployed as a set of Kafka Connect-compatible connectors, so you first need to define a MySQL connector configuration and then start the connector by adding it to Kafka Connect.
+
+{{< note >}}
+Currently, Materialize only supports Avro-encoded Debezium records. If you’re interested in JSON support, please reach out in the community Slack or leave a comment on [this GitHub issue](https://github.com/MaterializeInc/materialize/issues/5231).
+{{</ note >}}
+
+1. Create a connector configuration file and save it as `register-mysql.json`:
+
+    ```json
+    {
+      "name": "your-connector",
+      "config": {
+          "connector.class": "io.debezium.connector.mysql.MySqlConnector",
+          "tasks.max": "1",
+          "database.hostname": "mysql",
+          "database.port": "3306",
+          "database.user": "user",
+          "database.password": "mysqlpwd",
+          "database.server.id":"223344",
+          "database.server.name": "dbserver1",
+          "database.history.kafka.bootstrap.servers":"kafka:9092",
+          "database.history.kafka.topic":"dbserver1.history",
+          "database.include.list": "db1",
+          "table.include.list": "table1",
+          "include.schema.changes": false
+        }
+    }
+    ```
+
+    You can read more about each configuration property in the [Debezium documentation](https://debezium.io/documentation/reference/connectors/mysql.html#mysql-connector-properties). By default, the connector writes events for each table to a Kafka topic named `serverName.databaseName.tableName`.
+
```

Comment:
Can you invert the order of the sentences and make each its own paragraph? I want to break them up because they're doing two different things (one is pointing to reference doc and one is explaining the result of the actions you took).

Reply:
I'll just move it to the step below, it actually makes more sense there.